junit/MatrixTest.java

```diff
@@ -71,11 +72,10 @@ public void tooShort(){
     //Matrix(double[] vals, int m){}
     
     /**
-     * testMatrixConstructor3 -Tests the third matrix constructor as specified in issue #7.
+     * testMatrixConstructor3a -Tests the third matrix constructor with valid parameters as specified in issue #7.
```

Comment:
I think that's a good idea.  Not completely sure. 
An alternative would be to checkout that file from that branch.
I am learning so much about things we should have planned out before we started.
The branch structure is a bit of a mess, and we really need to take care of the doubled source files.